conans/client/runner.py

```diff
@@ -46,7 +46,7 @@ def __call__(self, command, output, log_filepath=None, cwd=None):
     def _pipe_os_call(self, command, stream_output, log_handler, cwd):
 
         try:
-            proc = Popen(command, shell=True, stdout=PIPE, stderr=PIPE, cwd=cwd)
+            proc = Popen(command, shell=True, stdout=PIPE, stderr=STDOUT, cwd=cwd)
```

Comment:
Please, add a comment to not forget the trick, stderr is going to STDOUT and stout to PIPE, so only one "get_stream_lines" are needed.